lib/components/canvas/_canvas_painter.dart

```diff
@@ -56,7 +56,7 @@ class CanvasPainter extends CustomPainter {
           needToRestoreCanvasLayer = true;
         }
         if (currentSelection?.strokes.contains(stroke) ?? false) {
-          paint.color = primaryColor;
+          paint.color = Color.lerp(stroke.strokeProperties.color.withInversion(invert), Colors.black, 0.25) ?? primaryColor;
```

Comment:
The `??` isn't necessary since neither argument is null. We can just use a `!` here and in the other line